cmd/crc/cmd/status.go

```diff
@@ -89,19 +86,29 @@ func (s *status) prettyPrintTo(writer io.Writer) error {
 	}
 	w := tabwriter.NewWriter(writer, 0, 0, 1, ' ', 0)
 
-	lines := []struct {
+	// todo: replace this with Pair when switching to go@1.18
+	type line struct {
 		left, right string
-	}{
+	}
+	lines := []line{
 		{"CRC VM", s.CrcStatus},
-		{"OpenShift", openshiftStatus(s)},
-		{"Podman", s.PodmanVersion},
-		{"Disk Usage", fmt.Sprintf(
+	}
+
+	if s.OpenShiftVersion != "" {
```

Comment:
I heard rumors that with openshift preset you should ideally be able to do everything you can do with podman preset. In other words, I'd probably suggest keeping podman version in the status output under openshift preset. Maybe others can comment on this too.

Reply:
I remove version only if it empty, which means that podman/openshift not present in current preset.